rosdep/python.yaml

```diff
@@ -4499,6 +4511,10 @@ python-virtualenv:
   fedora: [python-virtualenv]
   gentoo: [dev-python/virtualenv]
   ubuntu: [python-virtualenv]
+python-vine-pip:
```

Comment:
As CI indicates there is an alphabetic ordering issue with this package. Please revise your PR to resolve it.